sympy/physics/continuum_mechanics/beam.py

```diff
@@ -435,32 +436,31 @@ def apply_support(self, loc, type="fixed"):
 
```

Comment:
It would be appropriate to add a new "Returns" section after the parameters section to show what is returned in the same format as the Parameters section.